singularity-eos/eos/eos_vinet.hpp

```diff
@@ -0,0 +1,374 @@
+//------------------------------------------------------------------------------
+// © 2021-2022. Triad National Security, LLC. All rights reserved.  This
+// program was produced under U.S. Government contract 89233218CNA000001
+// for Los Alamos National Laboratory (LANL), which is operated by Triad
+// National Security, LLC for the U.S.  Department of Energy/National
+// Nuclear Security Administration. All rights in the program are
+// reserved by Triad National Security, LLC, and the U.S. Department of
+// Energy/National Nuclear Security Administration. The Government is
+// granted for itself and others acting on its behalf a nonexclusive,
+// paid-up, irrevocable worldwide license in this material to reproduce,
+// prepare derivative works, distribute copies to the public, perform
+// publicly and display publicly, and to permit others to do so.
+//------------------------------------------------------------------------------
+
+#ifndef _SINGULARITY_EOS_EOS_VINET_HPP_
+#define _SINGULARITY_EOS_EOS_VINET_HPP_
+#define SINGULARITY_ENABLE_EXCEPTIONS
```

Comment:
I should explain in more detail. Defining this here pollutes all files that include this header file with this define. 

But perhaps worse, the reason it's not doing anything for you in tests is because the include guards in the `singularity_error` file mean that if a file included above yours in the test file includes `singularity_error` *without* this define, defining it here does nothing.